server/src/main/java/org/elasticsearch/search/aggregations/metrics/TDigestState.java

```diff
@@ -51,11 +54,12 @@ static Type valueForHighAccuracy() {
     private final Type type;
 
     /**
-     * @deprecated this method will be removed after all usages are replaced
+     * @deprecated No-op circuit-breaked factory for TDigestState. Used in _search aggregations.
+     *             Please use the {@link #create(CircuitBreaker, double)} method instead on new usages.
```

Comment:
In a follow-up can we remove this or rename it to `createWithoutCircuitBreaking` or something?

Reply:
Sure! Now that those _will stay_, it makes all the sense. I'll rename them now to trigger the auto-merge 😈